api/src/test/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceTest.java

```diff
@@ -226,6 +228,16 @@ public DbUserAccessTier addUserToTier(DbUser user, DbAccessTier tier) {
             .setLastUpdated(Timestamp.from(Instant.now())));
   }
 
+  @Transactional
```

Comment:
Does this work here? I thought it only worked in Spring-managed beans?

Reply:
No idea!

Reply:
It's the pattern this file users and I don't understand it enough to change it.